runtime/assembly/api/block-builder.ts

```diff
@@ -16,28 +16,30 @@ export function BlockBuilder_apply_extrinsics(data: i32, len: i32): u64 {
 }
 
 /**
- * 
+ * In this iteration returns the instanciated InherentData instance as a u8 array
```

Comment:
*it does not return the Instaciated inherent Data
It returns an empty array in the current implementation